sdks/python/apache_beam/pvalue.py

```diff
@@ -425,8 +441,7 @@ def _from_runtime_iterable(it, options):
 
   def _view_options(self):
     return {
-        'data': self._data,
-        # For non-fn-api runners.
+        'data': self._data,  # For non-fn-api runners.
```

Comment:
Are these whitespace changes coming from yapf? This particularly seems to have moved the comment to the line above it.

Reply:
Yeah, yapf did that. I'm not sure why. When I run "yapf -ir ." from the root python directory, a lot of files are re-formatted. Is this unexpected behavior? If it's not, I can create an MR with just YAPF changes to the entire apache_beam module. Thoughts?

Reply:
That is unexpected, yes.
Try running `tox -e py3-yapf`. Perhaps the yapf you have installed is a newer version than tox's (0.29.0).

Reply:
Some of the YAPF changes are fixed but a few whitespace changes remain that I'll look into soon.